core/gather/gatherers/bf-cache-failures.js

```diff
@@ -112,6 +113,10 @@ class BFCacheFailures extends FRGatherer {
       waitForLoadEvent(session, 0).promise,
     ]);
 
+    // DevTools e2e tests can sometimes fail on the next command if we progress too fast.
+    // The only reliable way to prevent this is to wait for an arbitrary period of time.
+    await new Promise(resolve => setTimeout(resolve, TEMP_PAGE_PAUSE_TIMEOUT));
```

Comment:
could this be folded into `waitForLoadEvent`'s `pauseAfterLoadMs` param above?

Reply:
Yeahhh I guess it can lol

I haven't seen this issue in a while so maybe don't need this patch anymore 🤷

Reply:
Speak of the fucking devil
https://github.com/GoogleChrome/lighthouse/actions/runs/4080247314/jobs/7032792999